### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by Tom1380** [microsoft/vscode-postgresql#84]
Last commit was more than a year ago.

**Comment by dpalmetzvbg** [comment]
Yes, it seems like. It is a pity, because this is the only VSCode extension for postgres that works with OS-Authentication (at least in a windows-environment) in an enterprise-environment. :-(

**Comment by magnocarvalho** [comment]
> Last commit was more than a year ago.

I came here today to do the same thing. 🤦‍♂️

**Comment by afreeland** [comment]
...bummer.  I've enjoyed using this extension but can definitely it was like minimum viable project but I thought it would get some love at some point.
